projects/step-core/src/lib/client/augmented/services/augmented-grid-service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
```

Comment:
@neogucky this is the workaround for the removed methods from the openapi.json

Reply:
and btw: this is the workaround for the method that is defined as returning `application/json` but returning plain text: https://github.com/exense/step-frontend/blob/8c27bb0877f2b95128ad87aabae9860e8edfa7c8/projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts#L27

Reply:
Sounds good. I would imagine that the fix for grid will probably be not what you expect but another openapi file or something like that.